orbit/cmd/desktop/desktop.go

```diff
@@ -271,9 +296,33 @@ func main() {
 				}
 				myDeviceItem.Enable()
 
-				if sum.Notifications.NeedsMDMMigration {
+				if runtime.GOOS == "darwin" &&
+					(sum.Notifications.NeedsMDMMigration || sum.Notifications.RenewEnrollmentProfile) &&
+					mdmMigrator.CanRun() {
+
+					// if the device is unmanaged or we're
+					// in force mode and the device needs
+					// migration, enable aggresive mode.
+					aggresive := sum.Notifications.RenewEnrollmentProfile ||
+						sum.Notifications.NeedsMDMMigration &&
+							sum.Config.MDM.MacOSMigration.Mode == fleet.MacOSMigrationModeForced
```

Comment:
Total nit but this doesn't seem like it's gofmt'd? (actually, might not be a nit as the lint CI might fail on this?)